samples/subsys/usb/cdc_acm/sample.yaml

```diff
@@ -22,6 +22,7 @@ tests:
       - nucleo_f413zh
       - mimxrt685_evk/mimxrt685s/cm33
       - mimxrt1060_evk/mimxrt1062/qspi
+      - samd21_xpro
```

Comment:
I'll recommend to add `same54_xpro` in the integration list too. These 2 boards covers all series.